composer.json

```diff
@@ -13,7 +13,13 @@
     "autoload": {
         "psr-4": {
             "SystemCtl\\": [
-                "src/",
+                "src/"
+            ]
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "SystemCtl\\Tests\\": [
```

Comment:
Yeah, I just like the consitency. We have it for the namespace in the non-dev autoload section, too.
I find it easier to spot for others that this is extendable to some extend. That is why I used the array syntax.